### ISSUE TITLE
No keyboard navigation support in course player

### ISSUE THREAD

**Issue by augustina-jpg** [augustina-jpg/scoopdope#1048]
The course video player cannot be controlled with the keyboard. Add keyboard shortcut support.

**Comment by Saber1Y** [comment]
@Saber1Y has applied to work on this issue as part of the **Stellar Wave Program**'s 9th wave.



> Hi i can do a great job in solving this issue please assign to me you won’t be disappointed/dissatisfied i’m asking for your trust maintainer


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/75be4916-6e7f-4dbb-883d-c0d3ba40459c) or assign @Saber1Y to this issue.

**Comment by griffin-de-dev** [comment]
@griffin-de-dev has applied to work on this issue as part of the **Stellar Wave Program**'s 9th wave.



> I would like to work on this issue 


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/75be4916-6e7f-4dbb-883d-c0d3ba40459c) or assign @griffin-de-dev to this issue.
